src/parser/SparqlParser.cpp

```diff
@@ -159,24 +155,23 @@ void SparqlParser::parseQuery(ParsedQuery* query, QueryType queryType) {
 
 // _____________________________________________________________________________
 void SparqlParser::parseSelect(ParsedQuery* query) {
-  query->_clause =
-      parseWithAntlr(sparqlParserHelpers::parseSelectClause, *query);
+  query->_clause = parseWithAntlr(&AntlrParser::selectClause, *query);
   lexer_.expect("where");  // Still parsed with old parser. Expects WHERE
                            // keyword to be consumed.
 }
 
+// _____________________________________________________________________________
 // Helper function that converts the prefix map from `parsedQuery` (a vector of
```

Comment:
If you make it the `static` function (which is probably cleaner here),
the move the comment to the header, and leave the `// ________________` line here.